tests/conftest.py

```diff
@@ -6,12 +6,12 @@
 import pytest
 from pubsub import pub
 
-from finesse.hardware.serial_device import SerialDevice
-
 
 @pytest.fixture
 def serial_mock(monkeypatch) -> MagicMock:
     """Fixture for Serial's constructor."""
+    from finesse.hardware.serial_device import SerialDevice
```

Comment:
This outside of the function were preventing the tests to be loaded because of the error mentioned in the PR description. Now they are loaded fine, just most of those related to hardware fail in MacOS.